content/zh-cn/docs/contribute/style/diagram-guide.md

```diff
@@ -509,11 +526,11 @@ the `.svg` image file and add a caption.
 `.svg` 图片文件，并为之添加标题。
 
 <!--
-```json
+```text
 {{</* figure src="/static/images/boxnet.svg" alt="Boxnet figure" class="diagram-large" caption="Figure 14. Boxnet caption" */>}}
 ```
 -->
-```json
```

Comment:
Changing from `json` (the content is not json I know) to `text` removes some nice highlighting.
Before:
![image](https://github.com/user-attachments/assets/075028f9-5b41-4100-931f-627bb7f9382e)
After:
![image](https://github.com/user-attachments/assets/d79eab10-f1b2-413e-b289-8e761284e248)

The `en` version still uses `json` though.

Reply:
Yes, we can improve the consistency in the en version indeed.